packages/sanity/src/core/studio/components/navbar/search/contexts/search/SearchProvider.tsx

```diff
@@ -24,15 +25,30 @@ import {initialSearchState, searchReducer} from './reducer'
 interface SearchProviderProps {
   children?: ReactNode
   fullscreen?: boolean
+  /**
+   * if provided, then it means that the search is being done from within a specific perspective
+   */
+  perspective?: string
+  /**
+   * list of document ids
+   * if provided, then ids should be checked against this list
+   */
+  documentIds?: string[]
```

Comment:
Wdyt about renaming this to `excludedDocumentIds` or `disabledDocumentIds`.

Reply:
Ah this actually makes a lot of sense, I'll rename it!

Reply:
Done on this https://github.com/sanity-io/sanity/pull/7815/commits/b8567687a7f20148fc2b3ad99f3f71543ae540d3 :)